modules/l4xlb/variables.tf

```diff
@@ -34,3 +34,12 @@ variable "name" {
   description = "External LB name."
   type        = string
 }
+
+variable "labels" {
+  type        = map(string)
+  default     = {}
+  description = <<-EOD
```

Comment:
Style fix was required for the `heredocs` to follow the style guidelines. The same has been applied and pushed.

Reply:
thank you! 👍